pkg/wuffs/build.zig

```diff
@@ -12,9 +12,11 @@ pub fn build(b: *std.Build) !void {
     });
 
     const unit_tests = b.addTest(.{
-        .root_source_file = b.path("src/main.zig"),
-        .target = target,
-        .optimize = optimize,
+        .root_module = b.createModule(.{
+            .root_source_file = b.path("src/main.zig"),
+            .target = target,
+            .optimize = optimize,
```

Comment:
To be consistent with the others, perhaps `.target` and `.optimize` shouldn't be moved to the `createModule` call and kept with the `addTest` call?

Reply:
Whoops! Looks like it's the opposite. `.target` and `.optimize` need to be moved to `.createModule`.